src/php/gov-harvester/govharvester.php

```diff
@@ -169,10 +264,25 @@ function docParsePage($domdoc)
 			// write line to output file
 			fputcsv($fp,$csvdata);
 
+			// download the file
+			downloadfile($link[0]->getAttribute('href'));
+
+
 			continue;
 		}
 	}	
 }
 
+function downloadfile($path)
+{
+	global $confarray;
+
+	// construct a sensible filename
+	$filenumber = substr(strrchr(dirname($path),'/'),1);
+
+	// download the file
+	file_put_contents($confarray['destinationdir']. '/' . $filenumber . '_' . basename($path), fopen("https://www.gov.uk" . $path, 'r'));
+}
+
 
-?>
+php?>
```

Comment:
> The closing ?> tag MUST be omitted from files containing only PHP.

Closing `php` tag not required

Reference: http://www.php-fig.org/psr/psr-2/#2-2-files